### DISCUSSION TITLE
shared spack installation / serving applications to users

### DISCUSSION THREAD

**Discussion by ltalirz** [Azure/az-hop#999] [Ideas]
The [az-hop docs](https://azure.github.io/az-hop/tutorials/spack.html) documents how to set up a spack installation for a single user.

However, az-hop administrators will often want to provide pre-built software to their users.
In some cases, this software may be included directly in the cluster images, but in other cases a large number of packages may be needed that would blow up the image size (and may need updates frequently).
I.e. admins may instead want to point users to a shared spack installation on the ANF share or expose spack build artifacts via lmod modules, ...

It would be great to have a few pointers in the docs on best practices for how to go about this from the perspective of a site admin

**Comment by xpillons** [comment]
this blog should help you I guess https://techcommunity.microsoft.com/t5/azure-global/spack-in-a-multi-user-hpc-environment-on-azure/ba-p/3438261
